edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -135,12 +135,12 @@ public async Task SendMessageAsync(IMessage inputMessage)
             this.timer.Reset();
             try
             {
-                using (Metrics.TimeMessageSend(this.clientId))
+                using (Metrics.TimeMessageSend(this.clientId, inputMessage.ProcessedPriority))
```

Comment:
Do we need Priority for the message process time?

Reply:
hmm...I'm fine either way.  I just added it because it was relevant to messages, but one could argue that all messages are treated the same once they reach this point anyways.

---
In reply to: [397345125](https://github.com/Azure/iotedge/pull/2730#discussion_r397345125) [](ancestors = 397345125)